ParisBNF/abb/BNFabb72.xml

```diff
@@ -29,22 +32,607 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>                                                                       
+                           <note>The leaves <locus target="#i #3 #ii #iii"/> are neither pricked nor ruled.</note>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <locus from="4ra" to="178va"/>                  
+                        <seg type="script">Second half of the 17th century.</seg>                          
+                        <seg type="ink">Black; red.</seg>                     
+                        <desc>Careful, regular; by a good scribe. Possibly more than one scribe took 
+                           part in copying.</desc>
+                     </handNote>
+                     
+                     <handNote xml:id="h2" script="Ethiopic" corresp="#a4 #a5 #a6 #ms_i1.6.15">                                  
+                        <seg type="script">Second half of the 17th century/early 18th century.</seg>                          
+                        <seg type="ink">Black; red.</seg>                     
+                        <desc>Careful, somewhat irregular; by a mediocre scribe. The script is of smaller size, especially the one 
+                           used for additional notes.</desc>
+                     </handNote>
+                  </handDesc>     
+                      
+               
+               <additions>
+                  <list>                     
+                        
+                        <item xml:id="a1">
+                           <locus target="#1r"/>
+                           <desc type="ScribalNoteCommencing">Scribal note on commencing the writing</desc>
+                           <q xml:lang="gez">ወተወጥነ፡ ዛቲ፡ መጽሐፍ፡ አመ፡ <date>፮፡ ለመጋቢት፡</date> በዕለተ፡ ሐሙስ፡ በ፪፡ ሠርቀ፡ ሌሊት፡ 
+                              <gap reason="ellipsis" resp="DN"/></q>
+                           <note>Written possibly in <ref target="#h1">Hand 1</ref>.</note>
+                        </item>
+                    
+                     <item xml:id="a2">
+                        <locus from="178va" to="178vb"/>
+                        <desc type="GuestText"><title type="incomplete" ref="LIT1104Anapho"/>An excerpt from the Anaphora of the Apostles 
+                           ('Laying of hands')</desc>
+                        <q xml:lang="gez">አንብሮ፡ እድ፡ ዘቅዳሴ፡ ሐዋርያት፨ እግዚአብሔር፡ ዘለዓለም፡ ብርሃነ፡ ሕይወት፡ ዘኢይጠፍ፡ 
+                           <gap reason="ellipsis" resp="DN"/></q>
+                     </item> 
+                     
+                     <item xml:id="a3">
+                        <locus from="179ra" to="179rb"/>
+                        <desc type="Genealogy">A genealogy called in the beginning pen trial</desc>
```

Comment:
I don't understand what is meant by "called"

Reply:
It is a genealogy, but the scribe wrote in the beginning ፈቲነ ብርዕ

Reply:
Maybe it can be reformulated to make this clearer ("The words ፈቲነ ብርዕ, followed by a genealogy" or something like that)?

Reply:
ok, I have reformulated, a little different.